server/routes/creatingLicences/handlers/caseload.ts

```diff
@@ -28,9 +29,8 @@ export default class CaseloadRoutes {
     const { user } = res.locals
     const { prisonerNumber } = req.body
 
-    // TODO: This block is temporary, remove this when we have a design for editing existing licences
-    const comLicenses = await this.licenceService.getLicencesByStaffIdAndStatus([], user)
-    const existingLicense = comLicenses.find(licence => licence.nomisId === prisonerNumber)
+    // TODO: Which statuses should be considered "existing"? Should INACTIVE and RECALLED be included?
+    const existingLicense = _.head(await this.licenceService.getLicencesByNomisIdsAndStatus([prisonerNumber], [], user))
     if (existingLicense) {
```

Comment:
Just spelling `licence` not `license` :-)

Reply:
You'd think I'd know that by now!